examples/terraform-modules/teleport-azure-discovery/examples/single-subscription/main.tf

```diff
@@ -0,0 +1,20 @@
+terraform {
+  required_version = ">= 1.6.0"
+}
+
+module "teleport_azure_discovery" {
+  source = "../.."
+
+  subscription_id                = var.subscription_id
```

Comment:
should we support multiple subscriptions or even wildcard?

Reply:
as far as multiple subscruption IDs goes, im unsure how it would work atm. perhaps we go with single sub now and then expand when we do multi-account stuff with azure?

ill look into wildcards for regions/resource groups